test/resources/lcmap-landsat.edn

```diff
@@ -27,6 +27,7 @@
  :worker    {:exchange "test.lcmap.landsat.worker.exchange"
              :queue    "test.lcmap.landsat.worker.queue"}
  :search     {:index-url      "http://localhost:9200/tile-specs"
+              :refresh-url    "http://localhost:9200/tile-specs/_refresh"
```

Comment:
Convention-over-configuration might be nice for ES' refresh/bulk/search endpoints; fine like this for now though.